django_project/ta_system/admin.py

```diff
@@ -26,35 +26,42 @@
 
 class CustomAdminSite(AdminSite):
 
-    site_title  = "Boston College TA Application System"
+    site_title = "Boston College TA Application System"
     site_header = "Boston College TA Application System"
     index_title = "System Admin"
 
     def get_urls(self):
         urls = super().get_urls()
         urls = [
             path('', self.admin_view(self.index)),
-            path('course_data_upload', 
-                self.admin_view(self.course_data_upload), 
-                name='course_data_upload'),
-            path('applicant_data_upload', 
-                self.admin_view(self.applicant_data_upload), 
-                name='student_data_upload'),
+            path('course_data_upload',
+                 self.admin_view(self.course_data_upload),
+                 name='course_data_upload'),
+            path('applicant_data_upload',
+                 self.admin_view(self.applicant_data_upload),
+                 name='student_data_upload'),
             path('assignment_data_download',
-                self.admin_view(self.assignment_data_download),
-                name='assignment_data_download')
+                 self.admin_view(self.assignment_data_download),
+                 name='assignment_data_download'),
+            path('change_system_status',
+                 self.admin_view(self.change_system_status),
+                 name='change_system_status')
         ] + urls
         return urls
 
     def index(self, request):
         app_list = self.get_app_list(request)
+        status_list = models.SystemStatus.objects.order_by('id')
```

Comment:
Same question here as for the change_system_status view below, regarding whether or not we can order_by the 'date_changed' field instead of the 'id' field.

Reply:
I think order by id is the best because the id is assigned based on the most recent instance created. I am afraid that order by data_changed will mix up the status if I changed the status several times on the same date

Reply:
I'm pretty sure timezone.now and datetime.now both track down to milliseconds, so this wouldn't be a problem. But nevertheless it's fine leaving it as order_by('id').